main/index.js

```diff
@@ -59,6 +59,12 @@ log.info('Machine spec: %s version %s', os.type(), os.release())
 // log.info('Machine spec: %s %s version %s', os.type(),
 //   os.machine(),
 //   os.release())
+if (app.runningUnderARM64Translation) {
+  log.warn(
+    'Warning: we are running under ARM64 translation' +
+      ' (macOS Rosetta or Windows WOW).'
+  )
+}
```

Comment:
```suggestion
```

Do you think this log message is important? Our logs are quite noisy already, and we aren't doing anything in the electron process that is CPU intensive

Reply:
I added this log message to help us with troubleshooting. Maybe I can rephrase it from a warning to an informational message?

Reply:
Ok I can see that it's useful now. Warning sgtm